File: mobile/src/main/java/com/chernowii/camcontrol/camera/goproAPI/model/media/GoProFile.java
package com.chernowii.camcontrol.camera.goproAPI.model.media;

import com.google.gson.annotations.SerializedName;

public class GoProFile {

    @SerializedName("n")
    public String filename;

    @SerializedName("mod")
    public String mod;

    @SerializedName("s")
    public String filesize;
}

File: mobile/src/main/java/com/chernowii/camcontrol/camera/goproAPI/model/media/GoProFile.kt
package com.chernowii.camcontrol.camera.goproAPI.model.media

import com.google.gson.annotations.SerializedName

class GoProFile {

    @SerializedName("n")
    var filename: String? = null

    @SerializedName("mod")
    var mod: String? = null

    @SerializedName("s")
    var filesize: String? = null
}
